.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,20 @@
+name: SonarCloud Scan
+on:
+  push:
+    branches:
+      - next
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
This seems to be the default if `types` are not specified. Is there any particular reason for setting this explicitly anyway?

<img width="741" alt="image" src="https://user-images.githubusercontent.com/154725/196943028-91663e7d-1c56-4fe4-a1e5-52319c55606f.png">

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
Good to know: The whole workflow file is generated by Sonarcloud for you when setting up the CI mode. I've just copy pasted it. It doesn't hurt to be explicit with the event types!